apps/server/src/modules/tool/school-external-tool/service/school-external-tool.service.ts

```diff
@@ -57,20 +57,33 @@ export class SchoolExternalToolService {
 		tool: SchoolExternalTool,
 		externalTool: ExternalTool
 	): Promise<ToolConfigurationStatus> {
+		const configurationStatus: ToolConfigurationStatus = new ToolConfigurationStatus({
```

Comment:
Maybe you could use a better variable name

Reply:
toolConfigStatus ok ?